scvi/distributions/_negative_binomial.py

```diff
@@ -412,8 +413,8 @@ class ZeroInflatedNegativeBinomial(NegativeBinomial):
     arg_constraints = {
         "mu": constraints.greater_than_eq(0),
         "theta": constraints.greater_than_eq(0),
-        "zi_probs": constraints.half_open_interval(0.0, 1.0),
         "zi_logits": constraints.real,
+        "scale": constraints.greater_than_eq(0),
```

Comment:
does this force us to always provide scale?

Reply:
I don't think so, `test_distributions.py` runs smoothly even though `scale` is not provided.